android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -115,6 +126,28 @@ public Snapshot(View view) {
     }
     originXByParent = view.getLeft();
     originYByParent = view.getTop();
+    borderRadius = getBorderRadius(view);
+  }
+
+  private float getBorderRadius(View view) {
+    if (view.getBackground() != null) {
+      Drawable background = view.getBackground();
+      if (background instanceof ReactViewBackgroundDrawable) {
+        return ((ReactViewBackgroundDrawable) background).getFullBorderRadius();
+      }
+    } else if (view instanceof ReactImageView) {
+      try {
+        Field field = view.getClass().getDeclaredField("mBorderRadius");
+        field.setAccessible(true);
+        Float borderRadius = field.getFloat(view);
+        return Float.isNaN(borderRadius) ? 0 : borderRadius;
+      } catch (NullPointerException | NoSuchFieldException | IllegalAccessException ignored) {
+        // In case of non-standard view is better to not support the border animation
+        // instead of throwing exception
+      }
+    }
+    Log.w("[Reanimated]", "Unable to get border radius value for view: " + view);
```

Comment:
not sure about this warning here. It looks like we try to unconditionally extract border radius, which means people will get this warning for unsupported view types even when not trying to animate the border

Reply:
Resolved in https://github.com/software-mansion/react-native-reanimated/pull/4103/commits/8d40ceced834f3b82b8fab7cc43c5828cabf77ea